src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -2734,7 +2740,7 @@ namespace WaterToAirHeatPumpSimple {
 
                 // user provided inputs are assumed to be at rated conditions
                 simpleWatertoAirHP.RatedPowerHeat = simpleWatertoAirHP.RatedCapHeat / simpleWatertoAirHP.RatedCOPHeatAtRatedCdts;
-                simpleWatertoAirHP.RatedCapHeatAtRatedCdts = 0;
+                simpleWatertoAirHP.RatedCapHeatAtRatedCdts = simpleWatertoAirHP.RatedCapHeat;
```

Comment:
Why would the capacity be set to 0 if not autosized? And what about power? and check the cooling coil code.

    Real64 RatedCapHeatAtRatedCdts = 0.0;        // Rated Heating Capacity at Rated Conditions [W]

Reply:
I guess I could have changed the water flow rate sizing to use RatedCapHeat instead.

    if (simpleWatertoAirHP.WAHPType == WatertoAirHP::Heating) {
        RatedWaterVolFlowRateDes =
            simpleWatertoAirHP.RatedCapHeatAtRatedCdts / (state.dataSize->PlantSizData(PltSizNum).DeltaT * Cp * rho);

Reply:
There is likely a difference between RatedCapHeat and RatedCapHeatAtRatedCdts give the CapFT term applied. This is above at line 2743.

                simpleWatertoAirHP.RatedCapHeat = RatedCapHeatDes;
                simpleWatertoAirHP.RatedCapHeatAtRatedCdts = RatedCapHeatDes * RatedHeatCapTempModFac;

Reply:
I think these equations need to be moved up into the IF blocks in order to use the correct variable. If RatedCapHeat is the coil load and RatedCapHeatatRatedCdts is the coil size then declaring these variables AFTER the coil size is determined seems wrong (i.e., after the if/else). If autosized, RatedCapHeatatRatedCdts should be used as the capacity, if not autosized, RatedCapHeat should be used as the capacity.

Reply:
I actually want to revert this change and change the RatedWaterVolFlowRateDes sizing equation to use RatedCapHeat because that's what the sizing routine is setting as the final coil capacity. And the calc routine uses RatedCapHeat as the coil capacity. So RatedCapHeat should be the final coil size whether the coil is autosized or hard-sized.

    HeatCapAtPeak = rhoair * VolFlowRate * Psychrometrics::PsyCpAirFnW(DataPrecisionGlobals::constant_zero) *
                    (HeatSupTemp - HeatMixTemp); // heating coil load

    RatedCapHeatDes = (PeakHeatCapTempModFac > 0.0) ? HeatCapAtPeak / PeakHeatCapTempModFac : HeatCapAtPeak;

    // heating capacity final determination
    simpleWatertoAirHP.RatedCapHeat = RatedCapHeatDes;
    simpleWatertoAirHP.RatedCapHeatAtRatedCdts = RatedCapHeatDes * RatedHeatCapTempModFac;

    void CalcHPHeatingSimple(EnergyPlusData &state,
        HeatCapRated = simpleWatertoAirHP.RatedCapHeat;

So it follows that the water flow rate should use RatedHeatCap because that's the coil size. You can't see an issue in model performance when the water flow rate is sized with either of these variables because only exit water temp changes, not performance. You would have to really critique the exiting water temp to see an issue and even then it would be hard to see.

    if (simpleWatertoAirHP.WAHPType == WatertoAirHP::Heating) {
        RatedWaterVolFlowRateDes = 
            simpleWatertoAirHP.RatedCapHeatAtRatedCdts / (state.dataSize->PlantSizData(PltSizNum).DeltaT * Cp * rho);

Reply:
RatedCapHeatDes is the CapFT modified coil capacity, as is RatedCapHeat. The next line modifies this value by the rated CapFT, which should be 1 by definition (but I expect it's not 1). So now I'm back to using RatedCapHeat to size the water flow rate. This is so very confusing but I think I am more comfortable with this now. @mjwitte ?

simpleWatertoAirHP.RatedCapHeatAtRatedCdts = RatedCapHeatDes * RatedHeatCapTempModFac;